s3_upload/s3_upload.py

```
@@ -118,7 +118,7 @@ def parse_args() -> argparse.Namespace:
     return parser.parse_args()
 
 
-def upload_single_run(args):
+def upload_single_run(args) -> None:
```

Ruff (0.8.0)

Missing type annotation for function argument `args`

(ANN001)